packages-next/admin-ui/src/pages/ListPage/index.tsx

```diff
@@ -676,7 +676,7 @@ function ListTable({
           })}
         </tbody>
       </TableContainer>
-      <Pagination listKey={listKey} total={count} currentPage={currentPage} pageSize={pageSize} />
+      <Pagination list={list} total={count} currentPage={currentPage} pageSize={pageSize} />
```

Comment:
Seemed unnecessary for the Pagination component itself to be reaching into context for the list in this scenario. So i've passed the entire list to the component instead. 

Again happy to revert this if there's a good reason for doing this